metrics/exporter.go

```diff
@@ -94,6 +194,10 @@ func setCurMetricsConfig(c *metricsConfig) {
 // Return value indicates whether the exporter is flushable or not.
 func FlushExporter() bool {
 	e := getCurMetricsExporter()
+	return flushExporterHelper(e)
+}
+
+func flushExporterHelper(e view.Exporter) bool {
```

Comment:
Change to `flushExporterUnlocked` to keep consistent?

Reply:
I can do that. I was using "Helper" to indicate the function stub change, but Unlocked works too.